dev.yml

```diff
@@ -0,0 +1,28 @@
+name: argo-admin
```

Comment:
We only need this because of the playground, right?

Reply:
Yes, exactly